docs/src/pages/guides/composition/composition.md

```diff
@@ -10,9 +10,10 @@ To solve this problem we tag some of our components when needed
 with a `muiName` static property.
 
 However, users like to wrap components in order to enhance them.
-That can conflict with our `muiName` solution.
+That can conflict with our `muiName` solution. If you wrap a component verify if
```

Comment:
I have been avoiding the use of "users" (or worse "people") in the docs (and also possessives such as "we", and "our" where possible). Good practice in product documentation is to write as if speaking _to_ the user, so: 

"However, you may need to wrap a component in order to enhance it, which can conflict with the `muiName` solution."

Reply:
@mbrookes This is a great guideline. Should we write it somewhere or it's enough to have it implicit?

Reply:
If we can find a good public documentation style guide we could link to it from somewhere. I started to document conventions in the github wiki way back, but it was far from comprehensive. Better to point to a good resource than to try to reinvent the wheel.